paf-mvp-frontend/src/utils/cookie.ts

```diff
@@ -1,2 +1,8 @@
 export const getCookieValue = (name: string): string | undefined =>
   document.cookie.match(`(^|;)\\s*${name}\\s*=\\s*([^;]+)`)?.pop() || undefined;
+
+// default value : 24h
+export const DEFAULT_TTL_IN_SECONDS = 86400;
```

Comment:
[NIT] you might express this value with `24 * 3600` and the next one with `24 * 7 * 3600`, I think it would be a bit more readable, while as much efficient (no need to parse duration)